extensions/iobj-lib/standalone.mjs

```diff
@@ -87,6 +92,27 @@ class Standalone {
     return nextStateIncorrect;
   }
 
+  /**
+   * Show an open-ended dialog and check if the entered value is
+   * correct, interpreting the entered values as an totp.
+   */
+  async s2s_sf_showDialogAndCheckTOTP(srcExt, playerID, kwargs, sfInfo) {
+    const {nextState, nextStateIncorrect, dialog, secret, otpWindow} = kwargs;
+    const res = await this.helper.callS2cAPI(playerID, 'dialog',
+    'showDialogWithPrompt', 60*1000, sfInfo.visibleName, dialog);
```

Comment:
eslint would tell you that your indentation should be 8 whitespaces.

Reply:
Done.